virtcontainers/clh.go

```diff
@@ -436,7 +429,13 @@ func (clh *cloudHypervisor) addDevice(devInfo interface{}, devType deviceType) e
 			device: v.Name(),
 			mac:    v.HardwareAddr(),
 		})
-
+	case types.HybridVSock:
+		clh.Logger().WithField("function", "addDevice").Infof("Adding HybridVSock: path=%s, cid=%d, port=%d", v.UdsPath, v.ContextID, v.Port)
```

Comment:
Could you change the log calls to be "structured" by pulling out the relevant info into a set of fields:

```go
.WithFields(logrus.Fields{
    "function" : "addDevice",
    "path" : v.UdsPath,
    "cid" : v.ContextID,
    "port" : v.Port,
}).Info("Adding HybridVSock")
```

Reply:
Done.